app/page/postRank.js

```diff
@@ -0,0 +1,184 @@
+const nest = require('depnest')
+const { h, Value, computed, when } = require('mutant')
+const Scroller = require('pull-scroll')
+const next = require('pull-next-query')
+const pull = require('pull-stream')
+
+exports.gives = nest({
+  'app.html.menuItem': true,
+  'app.page.postRank': true
+})
+
+exports.needs = nest({
+  'app.html.scroller': 'first',
+  'app.sync.goTo': 'first',
+  'keys.sync.id': 'first',
+  'channel.obs.subscribed': 'first',
+  'contact.obs.following': 'first',
+  'sbot.pull.log': 'first',
+  'message.html.render': 'first'
+})
+
+exports.create = function (api) {
+  return nest({
+    'app.html.menuItem': menuItem,
+    'app.page.postRank': page
+  })
+
+  function menuItem () {
+    return h('a', {
+      style: { order: 1 },
```

Comment:
I'm merging this but know that christianbundy just went through removing all the order over-rides so he could sort them alphabetically